docs/ref/middleware.txt

```diff
@@ -495,6 +495,59 @@ Adds the ``user`` attribute, representing the currently-logged-in user, to
 every incoming ``HttpRequest`` object. See :ref:`Authentication in web requests
 <auth-web-requests>`.
 
+.. class:: LoginRequiredMiddleware
+
+    .. versionadded:: 5.1
+
+    Redirects all unauthenticated requests to a login page. For admin views,
```

Comment:
I know that we said in our sync today that classes have indented content in other docs, but this file does not seem to comply with that, since classes before this one does not indent their description:
![image](https://github.com/django/django/assets/124304/143b2e28-8212-45b5-9904-2de2dffc5a1e)

I *think* that we have to unindent everything up to the end to be consistent with the rest of the file, and later on consider a separated branch updating this docs to match other files, like everything under `class-based-views` which seems to indent all class definitions.